src/test/lambda/models/samLambdaRuntime.test.ts

```diff
@@ -123,3 +124,18 @@ describe('runtimes', function () {
         ])
     })
 })
+
+describe('getNodeMajorVersion()', () => {
+    it(`extracts major version from 'nodejs{VERSION}.x'`, () => {
+        for (let i = 1; i < 1000; i++) {
+            const versionString = `nodejs${i}.x`
```

Comment:
Sorry for the confusion. I was suggesting to use the `nodeJsRuntimes` list from `src/lambda/models/samLambdaRuntime.ts`. That way, as we add support for new runtimes, we have coverage for all new runtime names.

Reply:
Ah I see now, updated